lib/src/main/java/com/orange/ods/compose/component/button/OdsIconToggleButtonsRow.kt

```diff
@@ -0,0 +1,169 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.compose.component.button
+
+import androidx.compose.animation.animateColorAsState
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.IntrinsicSize
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.selection.selectableGroup
+import androidx.compose.material.Divider
+import androidx.compose.material.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.orange.ods.compose.component.OdsComponentApi
+import com.orange.ods.compose.component.utilities.DisabledInteractionSource
+import com.orange.ods.compose.component.utilities.Preview
+import com.orange.ods.compose.component.utilities.UiModePreviews
+import com.orange.ods.compose.theme.OdsDisplaySurface
+import com.orange.ods.compose.theme.OdsTheme
+
+/**
+ * <a href="https://system.design.orange.com/0c1af118d/p/06a393-buttons/b/79b091" target="_blank">ODS Buttons</a>.
+ *
+ * A group of toggle buttons. Only one option in a group of toggle buttons can be selected and active at a time.
+ * Selecting one option deselects any other.
+ *
+ * @param iconToggleButtons Contains the buttons to display in the toggle group
+ * @param selectedIndex The [iconToggleButtons] list index of the selected button.
+ * @param onSelectedIndexChange Callback to be invoked when the selection change.
+ * @param modifier optional [Modifier] for this IconToggleButton
+ * @param displaySurface optional allow to force the button display on a dark or light
+ * surface. By default the appearance applied is based on the system night mode value.
+ */
+@Composable
+@OdsComponentApi
+fun OdsIconToggleButtonsRow(
+    iconToggleButtons: List<OdsIconToggleButtonsRowItem>,
+    selectedIndex: Int,
+    onSelectedIndexChange: (Int) -> Unit,
+    modifier: Modifier = Modifier,
+    displaySurface: OdsDisplaySurface = OdsDisplaySurface.Default
+) {
+    Row(
+        modifier = modifier
+            .height(IntrinsicSize.Min)
```

Comment:
Just curious but without this line the dividers are not displayed. Do you know why ?